File: Android/Objects/src/com/apps/demo/designerscripts/LS_frmabout.java
package com.apps.demo.designerscripts;
import anywheresoftware.b4a.objects.TextViewWrapper;
import anywheresoftware.b4a.objects.ImageViewWrapper;
import anywheresoftware.b4a.BA;


public class LS_frmabout{

public static void LS_general(java.util.LinkedHashMap<String, anywheresoftware.b4a.keywords.LayoutBuilder.ViewWrapperAndAnchor> views, int width, int height, float scale) {
anywheresoftware.b4a.keywords.LayoutBuilder.setScaleRate(0.3);
//BA.debugLineNum = 2;BA.debugLine="AutoScaleAll"[frmabout/General script]
anywheresoftware.b4a.keywords.LayoutBuilder.scaleAll(views);
//BA.debugLineNum = 3;BA.debugLine="AutoScaleRate(0)"[frmabout/General script]
anywheresoftware.b4a.keywords.LayoutBuilder.setScaleRate(0d);
//BA.debugLineNum = 4;BA.debugLine="pnlloading1.HorizontalCenter = 50%x"[frmabout/General script]
views.get("pnlloading1").vw.setLeft((int)((50d / 100 * width) - (views.get("pnlloading1").vw.getWidth() / 2)));
//BA.debugLineNum = 5;BA.debugLine="pnlloading1.VerticalCenter = 38%y"[frmabout/General script]
views.get("pnlloading1").vw.setTop((int)((38d / 100 * height) - (views.get("pnlloading1").vw.getHeight() / 2)));

}
}

File: iOS/Objects/src/B4iProject/LS_frmabout.m

#import "iCore.h"
@interface LS_frmabout:NSObject
@end

@implementation LS_frmabout

- (void)LS_general:(B4ILayoutData*)views :(int)width :(int)height{
[B4ILayoutBuilder setScaleRate:0.3];
//BA.debugLineNum = 2;BA.debugLine="AutoScaleAll"[frmabout/General script]
[B4ILayoutBuilder scaleAll:views :width :height];
//BA.debugLineNum = 3;BA.debugLine="AutoScaleRate(0)"[frmabout/General script]
[B4ILayoutBuilder setScaleRate:0.0];
//BA.debugLineNum = 4;BA.debugLine="pnlloading1.HorizontalCenter = 50%x"[frmabout/General script]
[views get:@"pnlloading1"].Left = (int)((50 / 100.0 * width) - ([views get:@"pnlloading1"].Width / 2));
//BA.debugLineNum = 5;BA.debugLine="pnlloading1.VerticalCenter = 38%y"[frmabout/General script]
[views get:@"pnlloading1"].Top = (int)((38 / 100.0 * height) - ([views get:@"pnlloading1"].Height / 2));

}
@end